src/main/resources/helpTemplates/generic.template.html

```diff
@@ -0,0 +1,215 @@
+		</#if>
+		<#if arg.options?has_content>
+			<p>
+				The ${arg.name} argument is an enumerated type (${arg.type}), which can have one of the following values:
+			<dl class="enum">
+				<#list arg.options as option>
+					<dt class="enum">${option.name}</dt>
+					<dd class="enum">${option.summary}</dd>
+				</#list>
+			</dl>
+			</p>
+		</#if>
+		<p><#if arg.required != "NA">
+			<#if arg.required == "yes">
+				<span class="badge badge-important">R</span>
+			</#if>
+		</#if>
+			<span class="label label-info ">${arg.type}</span>
+			<#if arg.defaultValue?has_content>
+				&nbsp;<span class="label">${arg.defaultValue}</span>
+			</#if>
+			<#if arg.minValue?is_number>
+				&nbsp;<span class="label label-warning">[ [ ${arg.minValue}</span>
+			</#if>
+			<#if arg.minRecValue?is_number>
+				&nbsp;<span class="label label-success">[ ${arg.minRecValue}</span>
+			</#if>
+			<#if arg.maxRecValue?is_number>
+				&nbsp;<span class="label label-success">${arg.maxRecValue} ]</span>
+			</#if>
+			<#if arg.maxValue?is_number>
+				&nbsp;<span class="label label-warning">${arg.maxValue} ] ]</span>
+			</#if>
+		</p>
+	</#macro>
+
+	<#macro relatedByType name type>
+		<#list relatedDocs as relatedDoc>
+			<#if relatedDoc.relation == type>
+				<h3>${name}</h3>
+				<ul>
+					<#list relatedDocs as relatedDoc>
+						<#if relatedDoc.relation == type>
+							<li><a href="${relatedDoc.filename}">${relatedDoc.name}</a> is a ${relatedDoc.relation}</li>
+						</#if>
+					</#list>
+				</ul>
+				<#break>
+			</#if>
+		</#list>
+	</#macro>
+
+	<?php $group = '${group}'; ?>
+
+	<section class="span4">
+		<aside class="well">
+			<a href="index"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
+		</aside>
+		<aside class="well">
+			<h2>Categories</h2>
+			<@getCategories groups=groups />
+		</aside>
+		<?php getForumPosts( '${name}' ) ?>
+
+	</section>
+
+	<div class="span8">
+
+		<h1>${name}</h1>
+
+		<p class="lead">${summary}</p>
+
+		<#if group?? >
+			<h3>Category
+				<small> ${group}</small>
+			</h3>
+		</#if>
+		<hr>
+		<h2>Overview</h2>
+		${description}
+
+		<#-- Create references to additional capabilities if appropriate -->
+			<#if testPlugin?size != 0>
+				<hr>
+				<h2>Additional Information</h2>
+				<p></p>
+			</#if>
+			<#if testPlugin?size != 0>
+				<h3>Test Plugins</h3>
+				<#if testPlugin?size = 1>
+					<p>This Test Plugin is automatically applied to the data by the Engine before processing by ${name}.</p>
+				</#if>
+				<#if (testPlugin?size > 1) >
+					<p>These Test Plugins are automatically applied to the data by the Engine before processing by ${name}.</p>
+				</#if>
+				<ul>
+					<#list testPlugin as plugin>
+						<li><a href="${plugin.filename}">${plugin.name}</a></li>
+					</#list>
+				</ul>
+			</#if>
+			<#if extradocs?size != 0 || arguments.all?size != 0>
+				<hr>
+				<h2>Command-line Arguments</h2>
+				<p></p>
+			</#if>
+			<#if extradocs?size != 0>
+				<h3>Engine arguments</h3>
```

Comment:
You're right that we don't need this for engine args since they're determined by the "common" property, but I left this in the code since it seemed like a nice general mechanism for referencing additional docs (was this previously used exclusively for engine arguments ? it looks that way based on how it appears in the template.) However, the "Engine arguments" bit should be removed from the template here).